ads/springAds.md

```diff
@@ -44,18 +44,15 @@ limitations under the License.
 >
 </amp-ad>
 <amp-ad width="320" height="50" type="springAds" data-adslot="banner"></amp-ad>
-<amp-ad width="300" height="250" type="springAds" data-adslot="mrec"></amp-ad>
+<amp-ad width="100vW" height="fluid" type="springAds" data-adslot="mrec"></amp-ad>
```

Comment:
`100vw`

could you also update your example in ads.amp.html and make sure it's working?